pegasus/sites.v3/code.org/views/oceans_promo.haml

```diff
@@ -0,0 +1,51 @@
+%link{href: "/css/oceans-promo.css", rel: "stylesheet"}
+
+.img-container.desktop-feature
+  %img.background-img{src: '/images/fit-1940/oceans/oceans-background.png'}
+  %img.sea-creature{src: 'images/oceans/big-sunfish.png'}
+  %img.sea-creature{src: 'images/oceans/small-sunfish.png'}
+  %img.sea-creature{src: 'images/oceans/turtle.png'}
+  %img.sea-creature{src: 'images/oceans/gorgon-fish.png'}
+  %img.sea-creature{src: 'images/oceans/shark.png'}
+  %img.sea-creature{src: 'images/oceans/orange-fish.png'}
+  %img.crab{src: 'images/oceans/crab.png'}
+  .scanning-bot
+    %img.ai-bot{src: 'images/oceans/ai-bot.png'}
+  %img.ai-light{src: 'images/oceans/bot-beam-puddle.png'}
+  %img.ai-fish{src: 'images/oceans/fish-in-beam.png'}
+
+%img.mobile.mobile-feature{src: 'images/oceans/mobile-ai.png'}
+
+:javascript
```

Comment:
:speak_no_evil: JavaScript in HAML oh no!

Should be fine for this purpose as long as we guarantee jQuery will always be available in this context (which I'm pretty sure we do absolutely everywhere).